src/header/css/header-brand.pcss

```diff
@@ -46,6 +46,8 @@
 	fill: var(--mono-20);
 	height: 30px;
 	vertical-align: -6px;
```

Comment:
Did you try just change the `vertical-align` value to like `-4px` rather than adding an additional `top` value?

Reply:
Good catch, I was able to work it out with `vertical-align: -9px`... worked best. 
![image](https://user-images.githubusercontent.com/6144462/56768522-7bec0780-6763-11e9-8e58-19b9349f1da7.png)

Reply:
although, now we're scaling the pixel-perfect 32px icons to 30px.... 😬

Reply:
ah, yes. Updated. Since the size changed I had to update the vertical-align to `-10px`

![image](https://user-images.githubusercontent.com/6144462/56769129-26b0f580-6765-11e9-9412-ff7238d3bf92.png)